docs/source/api/v4/cdns_name_configs_monitoring.rst

```diff
@@ -27,6 +27,7 @@ Retrieves information concerning the monitoring configuration for a specific CDN
 
 :Auth. Required: Yes
 :Roles Required: None
+:Permissions Required: CDN:READ, SERVER:READ, TYPE:READ, PROFILE:READ
```

Comment:
seems like this endpoint has the same problems as snapshots, so you may just want to employ the same solution where you just give it its own Permission set and throw up your hands about all the data therein exposed.